dunstctl

```diff
@@ -95,7 +95,8 @@ case "${1:-}" in
 			|| die "Dunst controlling interface not available. Is the version too old?"
 		;;
 	"")
-		die "dunstctl: No command specified. Please consult the usage."
+		echo "dunstctl: No command specified. Please consult the usage."
+		show_help
```

Comment:
1. I'd prefer having a clear error message without the clutter of a long help page.
1. The exit code is now `0`.